src/graphnet/data/dataconverter.py

```diff
@@ -430,8 +430,16 @@ def _extract_data(self, fileset: FileSet) -> List[OrderedDict]:
             except:  # noqa: E722
                 continue
 
-            # Extract data from I3Frame
-            results = self._extractors(frame)
+            # Try to extract data from I3Frame else skip frame
```

Comment:
I figure we need to remove or change this, see comment in main thread on this PR.